src/main/java/com/google/finapp/SpannerDao.java

```diff
@@ -31,7 +31,8 @@
     this.databaseClient = databaseClient;
   }
 
-  void createCustomer(ByteArray customerId, String name, String address) throws SpannerException {
+  @Override
```

Comment:
Can we rename the class SpannerDao to convey that this implementation uses spanner's java client as opposed to jdbc.